data/human/human missions.txt

```diff
@@ -6890,3 +6890,109 @@ mission "Hogshead's Dilemma II"
 			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh while others seem disappointed or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - Shippy McShipface. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
 			`	All of the hyped discussions around Hogsheads in the past gave them a brief but strong popularity amongst transport organizations looking to grab the average passenger's attention. In the end, the company responsible for this naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back, along with the interest of several pilots. Perhaps the name "Shippy McShipface" is fitting after all.`
 				decline
+
+
+
+mission "Lost Dog"
+	name "Bring a dog to <planet>"
+	description "Bring Caesar, a dog you found lost on <source>, to their owner on <destination>."
+	to offer
+		random < 15
+	source "Calda"
+	destination "Hestia"
```

Comment:
"Pookie, Part 2" is also a mission involving a dog which you're returning to its owner on Hestia. Maybe this one should involve some other destination planet?

Reply:
The mission works as a way to give the player a glimpse into what life is like on Hestia. A brief window into the life of the super-rich who are said to live there. 
That is to say the main character of this mission is Hestia (and to a lesser degree its relationship with Calda). To move it to a different world would be essentially require writing a brand new mission from the ground up.

Reply:
Seems to me like you could set it on Martini instead without having to do any major changes, if it occurred before the bombings. Or Farseer. Or many of the other very rich paradise planets.

Reply:
I like the idea of a Martini mission, and I appreciate your point. This was written at a time before the other dog mission existed - hence why it's happened like this. But I still feel like it's too nested within the specific atmosphere of Hestia to be moved without a significant re-write, and I am not prepared to do that.

Reply:
I don't think that Hestia is particularly unique in having a clique of ultra-rich people. I'm also not sure how this mission is inextricable from being on Hestia; I could easily see the mission's destination being Glory, for instance. In my opinion, changing the origin of this mission doesn't actually require any significant rewrites.

Reply:
I won't stop you or anyone else if they want to make that change.